src/Api/EventApi.php

```diff
@@ -73,21 +100,57 @@ public function getList(Calendar $calendar = null, $filter = '', $orderBy = '')
             $params = ['query' => $params];
         }
 
-        $request = $this->guzzle->createRequest('GET', $url, $params);
-        $response = $this->guzzle->send($request);
+        return $this->requestEvents($url, $params);
+    }
 
-        if (200 > $response->getStatusCode() || 300 <= $response->getStatusCode()) {
-            throw new ApiErrorException($response);
+    /**
+     * Get the occurrences, exceptions, and single instances of events in a calendar view
+     *
+     * If a calendar is given, it fetches the events for this calendar ;
+     * otherwise, it takes the primary
+     *
+     * If no from is given, it fetches one year before now
+     * If no to is given, it fetches one year after now
+     *
+     * @param DateTime $from The date and time when the event starts.
+     * @param DateTime $to The date and time when the event ends.
+     * @param string $filter `$filter` query parameter to give to the request
+     * @param string $orderBy `$orderBy` query, to have an order of elements
+     *
+     * @see https://msdn.microsoft.com/office/office365/APi/calendar-rest-operations#GetCalendarView
+     */
+    public function getCalendarView(
+        Calendar $calendar = null,
+        DateTime $from = null,
+        DateTime $to = null,
```

Comment:
these shouldn't be null, this is not the scope of this adapter to make things such as "magically pick an arbitrary date"